---a/net/minecraft/block/BlockDoubleWoodSlab.java
+++b/net/minecraft/block/BlockDoubleWoodSlab.java
@@ -1,12 +1,11 @@
 package net.minecraft.block;
 
-public class BlockDoubleWoodSlab extends BlockWoodSlab
-{
+public class BlockDoubleWoodSlab extends BlockWoodSlab {
+
     private static final String __OBFID = "CL_00002112";
 
-    public boolean isDouble()
-    {
+    public boolean isDouble() {
         return true;
     }
-}
+}
